### PR TITLE
Remove: unused SafeMath.sol import

### PR DESCRIPTION
**Pull request by BunsDev**
Fixes Unused Import of SafeMath.sol

Remove unused SafeMath.sol import

### CODE REVIEW

**Review by frangio — APPROVED** [review]
Thank you @BunsDev!

**Comment by BunsDev** [comment]
my pleasure 🌙

**Comment by Miroslavecek** [comment]
diky diky

File: contracts/finance/PaymentSplitter.sol [hunk 1]
```diff
@@ -4,7 +4,6 @@ pragma solidity ^0.8.0;
 
 import "../utils/Address.sol";
 import "../utils/Context.sol";
-import "../utils/math/SafeMath.sol";
```
Comment (BigBlueHundids) [inline contracts/finance/PaymentSplitter.sol:7]: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/783ac759a902a7b4a218c2d026a77e6a26b6c42d/contracts/finance/PaymentSplitter.sol

**Review by BigBlueHundids — COMMENTED** [review]
>.

**Pull request merged**